tools/workspace/double_conversion/repository.bzl

```diff
@@ -20,6 +20,8 @@ def _impl(repository_ctx):
             "/usr/include/double-conversion",
             "include/double-conversion",
         )
+    elif os_result.is_manylinux:
+        build_flavor = "ubuntu"
```

Comment:
BTW, what's going on here? I didn't need to patch this either; is double-conversion not used directly with the various optimizers disabled? Should we preemptively add the proper symlink here, even though it isn't needed yet?